prepare-commit-msg.py

```
@@ -0,0 +1,231 @@
+
+  Remove timeouts which were used to mitigate the racing issue but are
+  obsolete now.
+    """
+
+
+@click.command()
+@click.option('--debug', is_flag=True, help='Enable debug logging.')
+@click.option('--model', default='google/gemini-flash-1.5-8b', help='Select AI model.')
+@click.option('--commit-msg-filename', required=True, help='Path to the commit message file.')
+@click.option('--open-source', is_flag=True, help='Send complete diff instead of just filenames')
+def main(debug, model, commit_msg_filename, open_source):
```

Ruff (0.8.2)

`main` is too complex (11 > 10)

(C901)